system/src/Grav/Console/Cli/InstallCommand.php

```diff
@@ -66,12 +66,11 @@ protected function serve()
         $this->destination = rtrim($this->destination, DS) . DS;
         $this->user_path = $this->destination . USER_PATH;
 
-        if (false === $this->isWindows()) {
-            $local_config_file = exec('eval echo ~/.grav/config');
-            if (file_exists($local_config_file)) {
-                $this->local_config = Yaml::parse($local_config_file);
-                $this->output->writeln('Read local config from <cyan>' . $local_config_file . '</cyan>');
-            }
+        $home_folder = getenv('HOME') ?: getenv('HOMEDRIVE') . getenv('HOMEPATH');
```

Comment:
Good idea, I've extracted the method to ConsoleTrait as you suggested, with the change that it returns the local file path if found.